check_dynamodb_access.py

```
@@ -0,0 +1,27 @@
+import boto3
+import logging
+import os
+from dotenv import load_dotenv
+
+# Load AWS credentials and region from .env file
+load_dotenv()
+
+logging.basicConfig(level=logging.INFO)
+logger = logging.getLogger(__name__)
+
+def check_dynamodb_access():
+    try:
+        region = os.getenv("AWS_REGION", "us-east-1")  # fallback to default
```

error out instead of fallbacks